### ISSUE TITLE
Typings of `Model.find*` don't return instance of that model by default

### ISSUE THREAD

**Issue by mchencb** [RobinBuschmann/sequelize-typescript#192]
```typescript
import { Model } from "sequelize-typescript";

declare class TestModel extends Model<TestModel> {
  prop: string;
}

TestModel.findById(1).then(m => m.prop); // error TS2339: Property 'prop' does not exist on type 'Model<{}>'
TestModel.findById<TestModel>(1).then(m => m.prop); // this is OK, but not convenient
```

`sequelize-typescript` version: 0.5.0
`typescript` version: 2.6.1

**Comment by RobinBuschmann** [comment]
@mchencb this is fixed with `sequelize-typescript@0.6.0-beta.4`

**Comment by mchencb** [comment]
Cool! Thanks
